elasticsearch/es_util.py

```diff
@@ -0,0 +1,41 @@
+import docker_util
+import gen_util
+import k8_util
+
+# es_util.py
+#
+# Utility functions for interacting with Google Kubernetes Engine
+# to configure Elasticsearch.
+
+
+def prepare_cluster(config, runtime):
+
+  # Prep the eck-operator image
+  docker_util.prepare_image("docker.elastic.co/eck/eck-operator:1.1.2", config)
+  docker_util.format_all_in_one_yaml(
+    "docker.elastic.co/eck/eck-operator:1.1.2", config)
+
+  # Prep the elasticsearch image
```

Comment:
We should think about how we will keep in sync the various files that hard-code the version (7.7.1).